components/aero_sensors/SConscript

```diff
@@ -0,0 +1,279 @@
+
+
+env["LINKFLAGS"] = [link_flags + common_flags]
+
+RTOS_INCLUDE_DIRS = [
+        RTOS_DIR.Dir("include"),
+        FREERTOS_DIR.Dir("Source/include/"),
+        FREERTOS_DIR.Dir("Source/portable/GCC/ARM_CM3/"),
+        CMSIS_DIR.Dir("CMSIS/Core/Include/"),
+]
+
+FS_INCLUDE_DIRS = [
+    FS_DIR.Dir("include"),
+    FATFS_DIR, # Necessary so that integer.h comes from FS/include
+    LIBS_DIR.Dir("FreeRTOS/CMSIS/RTOS2/FreeRTOS/Include1/"),
+    CMSIS_DIR.Dir("CMSIS/RTOS2/Include/"),
+]
+
+DRIVERS_INCLUDE_DIRS = [
+    chip_config["cmsis_includes"],
+    chip_config["hal_includes"],
+]
+
+LIBS_INCLUDE_DIRS = [
+]
+
+PROJECT_INCLUDE_DIRS = [
+    "include/",
+    "HW/include/",
+    "generated/",
+    #"lib/lib_FTDI_FT81x/include",
+]
+
+paths = (
+    FS_INCLUDE_DIRS + RTOS_INCLUDE_DIRS + DRIVERS_INCLUDE_DIRS + PROJECT_INCLUDE_DIRS + LIBS_INCLUDE_DIRS
+)
+
+env.Append(
+    ASFLAGS=as_flags,
+    CPPPATH=paths,
+    CCFLAGS=common_flags + c_flags,
+    LINKSCRIPT=chip_config["linker_file"],
+)
+
+rtos_env.Append(
+    CPPPATH=paths,
+    CCFLAGS=common_flags + c_flags + free_rtos_flags,
+)
+
+# List of component C files
+
+project_source_files = [
+    PROJECT_SOURCE_DIR.File("SystemManager.c"),
+    PROJECT_SOURCE_DIR.File("Module.c"),
+    PROJECT_SOURCE_DIR.File("Sensors.c"),
+    PROJECT_SOURCE_DIR.File("Files.c"),
+    PROJECT_SOURCE_DIR.File("IO.c"),
+]
+
+project_hw_files = [
+        PROJECT_HW_DIR.File("HW_adc.c"),
+        #PROJECT_HW_DIR.File("HW_can.c"),
+        PROJECT_HW_DIR.File("HW_dma.c"),
+        PROJECT_HW_DIR.File("HW_gpio.c"),
+        PROJECT_HW_DIR.File("HW_intc.c"),
+        PROJECT_HW_DIR.File("HW_msp.c"),
+        PROJECT_HW_DIR.File("HW_spi.c"),
+        PROJECT_HW_DIR.File("HW_timebase.c"),
+        PROJECT_HW_DIR.File("HW_i2c.c"),
+        PROJECT_HW_DIR.File("HW_MPRL.c"),
+        PROJECT_HW_DIR.File("HW_MAX7356.c"),
+        PROJECT_HW_DIR.File("HW_NPA.c"),
+        PROJECT_HW_DIR.File("HW_clock.c"),
+]
+
+ftdi_lib_files = [
+        #PROJECT_LIB_DIR.File("lib_FTDI_FT81x/src/EVE_commands.c"),
+        #PROJECT_LIB_DIR.File("lib_FTDI_FT81x/src/EVE_target.c"),
+]
+
+project_source_files += project_hw_files
+
+rtos_source_files = [
+        RTOS_DIR.File("FreeRTOSResources.c"),
+        RTOS_DIR.File("FreeRTOS_SWI.c"),
+        FREERTOS_DIR.File("Source/croutine.c"),
+        FREERTOS_DIR.File("Source/event_groups.c"),
+        FREERTOS_DIR.File("Source/list.c"),
+        FREERTOS_DIR.File("Source/queue.c"),
+        FREERTOS_DIR.File("Source/tasks.c"),
+        FREERTOS_DIR.File("Source/timers.c"),
+        FREERTOS_DIR.File("Source/portable/GCC/ARM_CM3/port.c"),
+        # FREERTOS_DIR.File("Source/stream_buffer.c"),
+        # FREERTOS_DIR.File("Source/portable/MemMang/heap_4.c"),
+]
+
+platform_source_files = [
+        # (
+        #     LIBS_DIR.File("printf/printf.c"),
+        #     ["-Wno-float-equal", "-DPRINTF_INCLUDE_CONFIG_H"],
+        # ),
+    (FS_DIR.File("fatfs.c"), ["",""]),
```

Comment:
Is this file just a copy of the one from the library?

Also, python files should be formatted with `black`. This seems wrong